manifests/j/juice/juice/0.3.5/juice.juice.locale.es-MX.yaml

```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.7-3-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: juice.juice
+PackageVersion: 0.3.5
+PackageLocale: es-MX
+Publisher: Atticus64
+PublisherUrl: https://github.com/Atticus64
+# PublisherSupportUrl:
+# PrivacyUrl:
+Author: Jonathan (Atticus64)
+PackageName: Juice
+PackageUrl: https://github.com/Atticus64/juice
+License: MIT
+LicenseUrl: https://raw.githubusercontent.com/Atticus64/juice/main/LICENSE
+# Copyright:
+# CopyrightUrl:
+ShortDescription: A fresh and sweet script to configure and personalize windows terminal
```

Comment:
Since this file contains only English, shouldn't it be named `juice.juice.locale.en-US.yaml`?

Reply:
oh right my mistake, in one moment i change it